schema/shell.json

```diff
@@ -0,0 +1,70 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://docs.oasis-open.org/openeox/tbd/schema/shell.json",
+    "title": "EOL and EOS Information",
+    "description": "A draft schema for representing End-of-Life (EOL) and End-of-Support (EOS) information in OpenEoX.",
+    "type": "object",
+    "$defs": {
+        "productName_t": {
+            "title": "Product name",
+            "description": "Contains the name of the product.",
+            "type": "string",
+            "minLength": 1
+          },
+          "productVersion_t": {
+            "title": "Product version",
+            "description": "Contains the version or release of the product.",
+            "type": "string",
+            "minLength": 1
+          },
+        "vendorName_t": {
+            "title": "Vendor name",
+            "description": "Contains the name of the vendor or service provider.",
+            "type": "string",
+            "minLength": 1
+        },
```

Comment:
It's great to see the spec progressed during VulnCon! :tada: 

I have a question: Will it be possible to specify the product in other ways? I think it would be nice to be able to define it with identifiers (purl, cpe, etc) or hashes (as in "the binary identified by hash X will be under support until YYYY).

Reply:
We are working on different approaches to identify products.

Regarding the "(as in "the binary identified by hash X will be under support until YYYY)": I currently doubt that this information will be actionable...

Reply:
No worries, we can still wrap the EoX Core data in an attestation and identify it cryptographically. I will formalize it with the intoto folks once this merges.

Reply:
That sound like the right approach. We indent to support multiple use cases for standalone OpenEoX information but definitely not all that are possibly out there.
So if you have a product identification scheme that works in your ecosystem, feel free to include the Core schema - that is exactly what it was build for: to be integrated in different environments.